src/http-libwebsockets.c

```diff
@@ -950,8 +950,8 @@ static void* http_thread_run(void* v) {
                 break;
             case W_WAKE: {
                 connection_t *c = w.value;
-                pn_collector_put(c->driver.collector, PN_OBJECT, c->driver.connection,
-                                 PN_CONNECTION_WAKE);
+                pn_collector_put_object(c->driver.collector, c->driver.connection,
+                                        PN_CONNECTION_WAKE);
```

Comment:
There should be update to CMakeLists.txt to set minimal supported version of Proton, at this line

https://github.com/skupperproject/skupper-router/blob/ae4e3c92c59c273ffaf3862bee724ad2801982f2/CMakeLists.txt#L116